.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.8
+    hooks:
+    -   id: flake8
+        args: ['--max-line-length=90']
+-   repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
```

Comment:
Black defaults to 88 but might get a bit longer if needed, there should be the least minimum arguments so that formatting is a no-brainer. The only tool that has an argument is Flake8 to accommodate Black defaults as explained in OP.